syntax = "proto3";

service ProcesadorImagen {
  rpc ProcesarImagen(ImagenRequest) returns (ImagenReply);
}

message ImagenRequest {
  bytes data = 1;
}

message ImagenReply {
  string status = 1;
  bytes image_data = 2;
}